hieradata/site/cp/role/htcondor.yaml

```diff
@@ -0,0 +1,19 @@
+---
+classes:
```

Comment:
It appears that `spec/unit/top_level_role_spec.rb` was broken by the migration to eyaml.  It should have failed on top level `htcondor` role not existing.  Our pattern is that there must be a top level role.  The classes/etc. should be factored out to a common top level role.

Reply:
Fix for the broken spec: https://github.com/lsst-it/lsst-control/pull/1118

Reply:
i need more specifics on this as i don't fully understand what this means, will ask later, thanks!

Reply:
`hieradata/role/htcondor.yaml` is required to exist.

Reply:
#1118 has been merged.  When this branch is rebased on `production`, there should be a test failure now.

Reply:
yeah it broke but i added the file and now its fine 👍🏻